dts/bindings/cpu/arm,cortex-m.yaml

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) 2021 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+# Common fields for ARM Cortex-M CPU
+
+include: cpu.yaml
+
+properties:
+    swo-ref-freq:
```

Comment:
What about changing the name of the property to `swo-ref-frequency`? Since we have `clock-frequency` for CPU clock.